docker-multinode/common.sh

```diff
@@ -598,3 +505,29 @@ kube::log::error() {
     echo "    $message" >&2
   done
 }
+
+# Configure cni plugins
+kube::multinode::start_configure_cni() {
+  mkdir -p /opt/cni/bin
+  mkdir -p /etc/cni/net.d
+  mkdir -p /var/run/flannel
+
+  curl -L https://github.com/containernetworking/cni/releases/download/v0.3.0/cni-v0.3.0.txz -O
```

Comment:
As discussed, `cni` v0.3 and the `10-containernet.conf` manifest below should be packaged inside the docker container instead.